packages/opentelemetry-exporter-zipkin/test/zipkin.test.ts

```diff
@@ -188,10 +186,10 @@ describe('ZipkinExporter', () => {
             annotations: [
               {
                 value: 'my-event',
-                timestamp: (startTime + 10) * MICROS_PER_MILLI,
+                timestamp: (startTime + 10) * 1e6,
```

Comment:
Is this micros per second? Could a const a be used here still?

Reply:
Sure, done.